aten/src/ATen/native/native_functions.yaml

```diff
@@ -4039,28 +4039,16 @@
 - func: __lshift__.Scalar(Tensor self, Scalar other) -> Tensor
   use_c10_dispatcher: full
   variants: method, function
-  dispatch:
-    CPU: legacy::cpu::_th_lshift
-    CUDA: legacy::cuda::_th_lshift
```

Comment:
Bring back these dispatch lines, but make them always call to `__lshift__` so like:

```
dispatch:
  CPU: __lshift__
  CUDA: __lshift__
```

Even better would be to have `__lshift__cpu` and `__lshift_cuda` and skip the second dynamic dispatch for the CUDA case but you don't have to do this if you don't want to (doing this leads to some copy pasting)

Reply:
Changed.